cmd/image-converter/image-converter.go

```diff
@@ -25,13 +27,39 @@ func main() {
 	}
 }
 
-func convert(srcVolPath, dstVolPath, sourceFormat, targetFormat string) error {
+func convert(srcVolPath, dstVolPath, srcFormat, dstFormat string) error {
+	err := qemuimgConvert(srcVolPath, dstVolPath, srcFormat, dstFormat)
+	if err != nil {
+		return err
+	}
+
+	// Copy dst over src
+	if isBlockDevice(srcVolPath) {
+		err = qemuimgConvert(dstVolPath, srcVolPath, dstFormat, dstFormat)
+		if err != nil {
+			return err
+		}
+	} else {
+		// Use mv for files as its faster than qemu-img convert
+		cmd := exec.Command("mv", dstVolPath, srcVolPath)
+		err := cmd.Run()
+		if err != nil {
+			return err
+		}
+
+		return nil
```

Comment:
if the don't have `return nil` in the block above, why having it here? I'd rather drop it (or add it to the block above and drop it below)

Reply:
bad copy paste...